src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -55,15 +56,18 @@ public void canCreateATitleLevelRequestForMultipleAvailableItemsAndAMatchingPick
     itemsFixture.basedUponDunkirkWithCustomHoldingAndLocation(holdings.getId(),
       null);
 
+    final String expectedPatronComment = "I need the book";
     JsonObject requestBody = createInstanceRequestObject(
       instanceMultipleCopies.getId(), requesterId, pickupServicePointId,
-      requestDate, requestExpirationDate);
+      requestDate, requestExpirationDate).put("patronComments", expectedPatronComment);
```

Comment:
Rather than mutating the produced JSON after the method, could an overload be introduced where patron comments can be provided.

```suggestion
      requestDate, requestExpirationDate, patronComments);
```

Reply:
I have introduced a builder, so the method call is replaced with the request builder.